concepts/advanced-enumeration/.meta/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "blurb": "Enumeration also works for hashes and can be nested.",
+  "authors": ["iHiD"],
```

Comment:
```suggestion
  "authors": ["angelikatyborska"],
```

Reply:
All of the text was copied from the exercise, which was written by Jeremy 🙂 I don't want to be blamed for the content 😁

Reply:
Ah, okay. Makes sense :D